olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -128,7 +128,7 @@ spec:
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
     ## Serverless functions
```

Comment:
Should this be Knative Functions as well?

Reply:
It does not hurt to fix it.